### DISCUSSION TITLE
Neovim tsserver autocomplete not working properly on JSX unless I use `sx` property

### DISCUSSION THREAD

**Discussion by bkerz** [mantinedev/mantine#3649] [Q&A] [answered]
Hi, I'm using Neovim version 0.8.3 with a Next project and obviously Mantine. I like the project but there's a little detail with the autocompletion. It works fine so far, but when I'm writing JSX it doesn't suggest the default JSX props like `onChange`, `onClick`, etc, the kind of props every component has unless I write the `sx` prop, after that the LSP knows what to suggest. Here are some images:

![image](https://user-images.githubusercontent.com/48168047/222432301-5bd90f18-3b8b-4ee1-b0bd-c2bba65257ff.png)
![image](https://user-images.githubusercontent.com/48168047/222432410-1496f145-790f-4ed4-8a00-cfeda38c4e00.png)

Another thing is that this is only happening with neovim, I tried with VSCode but everything works fine, I tried using the latest Node LTS version and the version `16.19.0` and the issue happens in both versions. The "test" was done in this project https://github.com/mantinedev/mantine-next-template

Lastly I would like to thank everyone that wants to help me with this in advance and ask for ways I could debug this issue by my own (because I know not everyone is interested in neovim)

**Comment by rtivital** [comment, marked as answer]
It appears to me that the issue is most likely in neovim – different editors implement tsserver differently and there is no way of telling what causes this exact issue without going through editor source code.

**Reply by bkerz** [reply]
Thanks for answering! Yeah, after a lot of debugging and reading other people's issues, the issue turned our to be my neovim configuration, there were a couple of things I had to update as well as adding a few configs I was missing. That completely solved the problem. Anyway, thanks for answering.
